src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -8,12 +8,14 @@
   private ArrayList<String> ingredients;
   private ArrayList<String> steps;
   private ArrayList<String> tags;
+  private boolean published;
   private String authorID;
   private String description;
   private String imageBlobKey;
   private String name;
-  private int popularity;
+  private long popularity;
   private long id;
+  private long publicRecipeID;
 
   // Recipe constructor for the recipe cards for which less information is needed
   public Recipe(long id, String name, String imageBlobKey, String description) {
```

Comment:
Do you still need this?

Reply:
yes that's what Mathew and I are using for all the recipe cards that aren't the ones in user recipes